### PR TITLE
Until the cert initialization is available for OS treat it as private

### PR DESCRIPTION
**Pull request by tmjd**
## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

**Review by rene-dekker — APPROVED** [review]
lgtm

File: pkg/components/enterprise.go [hunk 1]
```diff
@@ -180,5 +185,6 @@ var (
 		ComponentTigeraTypha,
 		ComponentTigeraCNI,
 		ComponentCloudControllers,
+		ComponentCSRInitContainer,
```
Comment (rene-dekker) [inline pkg/components/enterprise.go:188]: Won't this make it use the wrong registry?

**Review by doublek — APPROVED** [review]
LGTM

File: pkg/components/enterprise.go (hunk 1 above)
Comment (tmjd) [reply pkg/components/enterprise.go:188]: This doesn't have any effect on the registry used. In components/references.go is what controls which registry it comes from. This PR removes it from the switch case that removes it from the Calico registry (where the OS components are) so the default case hits which is the Tigera registry (quay.io or gcr.io for testing/hashreleases).

**Pull request merged**